README.md

```diff
@@ -16,5 +16,5 @@ npm install --save-dev webdash-travis-ci
 
 You need to edit your `webdash.json` file, and add the following values:
 
-* `travisToken` => You can get it from [here](https://travis-ci.org/profile)
-* `repositoryName` => Equivalent to `username/repository-name`; `jadsalhani/webdash-travis-ci`
+* `token` => You can get it from [here](https://travis-ci.org/profile)
```

Comment:
it's missing that these 2 need to go under `travis:`
how about we show it as an example: (and keep/fix your explanation)

```JSON
{
    "travis": {
          "token": "travis-token-here",
          "githubRepo": "username/repo"
     } 
}
```